tests/jose/jws/verifier.spec.ts

```diff
@@ -1,14 +1,133 @@
+import { expect } from 'chai';
+import { DidResolver } from '../../../src/did/did-resolver';
+import { GeneralJwsVerifier } from '../../../src/jose/jws/general/verifier';
+import { signers } from '../../../src/jose/algorithms';
+import sinon from 'sinon';
+
+const { secp256k1 } = signers;
+
 describe('GeneralJwsVerifier', () => {
+  afterEach(() => {
+    // restores all fakes, stubs, spies etc. not restoring causes a memory leak.
+    // more info here: https://sinonjs.org/releases/v13/general-setup/
+    sinon.restore();
+  });
   describe('getPublicKey', () => {
-    xit('throws an exception if publicKeyJwk isn\'t present in verificationMethod', () => {});
-    xit('throws an exception if DID could not be resolved', () => {});
-    xit('throws an exception if appropriate key isnt present in DID Doc', () => {});
-    xit('throws an exception if verificationMethod type isn\'t JsonWebKey2020', () => {});
-    xit('returns public key', () => {});
+    const buildDidResolutionResult = (
```

Comment:
since the change is isolated to the usage of the schema validator, we can test directly against the validator by adding a file under `https://github.com/TBD54566975/dwn-sdk-js/tree/main/tests/validation/json-schemas` called `jwk-verification-method.spec.ts`, so the test code is not bloated with all the code to stub the resolver.

Reply:
great suggestion. Moved tests to `verification-method.spec.ts`